var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -105,6 +105,8 @@ class Paraview(CMakePackage, CudaPackage):
     depends_on('double-conversion')
     depends_on('expat')
     depends_on('eigen@3:')
+    conflicts('target=ppc64le:', when='@3.3.0:3.3.4')
+
```

Comment:
The conflict is not in paraview but in the eigen package itself so needs to be moved into `/var/spack/repos/builtin/packages/eigen/package.py`

Reply:
Ok, would I just add line 108, above, directly somewhere in here then?

https://github.com/qwofford/spack/blob/7e34c28333c46349d54be1d511030f324f637ceb/var/spack/repos/builtin/packages/eigen/package.py#L7-L44

Maybe around line 30? Something else?